core/server/worker/src/main/java/alluxio/worker/block/DefaultBlockWorker.java

```diff
@@ -524,41 +526,47 @@ public static final class Metrics {
     /**
      * Registers metric gauges.
      *
-     * @param blockWorker the block worker handle
+     * @param blockworker the BlockWorker
      */
-    public static void registerGauges(final BlockWorker blockWorker) {
-      MetricsSystem.registerGaugeIfAbsent(
+    public static void registerGauges(final BlockWorker blockworker) {
+      CachedGauge<BlockWorkerMetrics> cache = new CachedGauge<BlockWorkerMetrics>(1000, TimeUnit.MILLISECONDS) {
```

Comment:
store this in a member variable `cache `

Reply:
why do we need to store this in a member variable? for possible future use?